addons/mail/static/src/components/partner_im_status_icon/partner_im_status_icon.scss

```diff
@@ -1,49 +1,12 @@
 // ------------------------------------------------------------------
 // Layout
 // ------------------------------------------------------------------
-
+$o-status-icon-size: 1.2em;
 .o_PartnerImStatusIcon {
-    display: flex;
-    flex-flow: column;
-
-    width: 1.2em;
-    height: 1.2em;
-    line-height: 1.3em;
```

Comment:
Are you sure you want to remove the `line-height` property? There's probably a reason why it is used (in a specific situation or for compatibility reasons).